.github/workflows/deploy.yml

```diff
@@ -0,0 +1,16 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install -r requirements.txt
+      - run: mkdocs gh-deploy --force
```

Comment:
Two things, do we want to be deploying on push? Second from what I read in mkdocs, since this is for an organization, you will want `--remote-branch master`.